index.coffee

```diff
@@ -1,3 +1,4 @@
+'use strict'
```

Comment:
`'use strict'` directive can be safely omitted only in ES6 modules or when CommonJS module consists only of `class` declarations.